ansible_builder/main.py

```diff
@@ -1,20 +1,25 @@
-import filecmp
 import os
-import shutil
 import textwrap
 import yaml
 
 from . import constants
 from .colors import MessageColors
 from .exceptions import DefinitionError
 from .steps import AdditionalBuildSteps, GalaxySteps, PipSteps, IntrospectionSteps, BindepSteps
-from .utils import run_command, write_file
+from .utils import run_command, write_file, copy_file
 from .requirements import sanitize_requirements
 import ansible_builder.introspect
 
 
-# Files that need to be moved into the build context
-CONTEXT_FILES = ['galaxy', 'python', 'system']
+# Files that need to be moved into the build context, and their naming inside the context
+CONTEXT_FILES = {
+    'galaxy': 'requirements.yml',
+    'python': 'requirements_user.txt',
+    'system': 'bindep_user.txt'
+}
+BINDEP_COMBINED = 'bindep_combined.txt'
+BINDEP_OUTPUT = 'bindep_output.txt'
+PIP_COMBINED = 'requirements_combined.txt'
```

Comment:
I changed the naming of most things that go into the build context folder.

Before, we accepted the naming from the user.... I'd rather not, because these go through our own phases of combination, sanitization, and transformation. I want these to be as clear as possible so that we can document them.